tests/internal/test_validator.py

```diff
@@ -108,3 +112,40 @@ def test_update_scores_based_on_thresholds() -> None:
     for metric, expected in expected_is_bad.items():
         assert scores[metric]["is_bad"] is expected
     assert all(scores[k]["score"] == raw_scores[k]["score"] for k in raw_scores)
+
+
+def test_prompt_tlm_with_message_history() -> None:
```

Comment:
add test to confirm there is no query rewriting happening, whenever this is the first user message

Reply:
add test to confirm that the primary `TrustworthyRAG.score(prompt, response)` call happens with `prompt` reflecting the full chat history, not with `prompt` reflecting the rewritten query.  

Confirm you are using this TLM utils method:
https://github.com/cleanlab/cleanlab-tlm/commit/a479e323943bc55fbd2d2eb4f6f6e77991960aa2

to turn the chat history into a prompt string.